frontend/resourceadm/utils/resourceListUtils/resourceListUtils.ts

```diff
@@ -0,0 +1,13 @@
+import type { ResourceListItem } from 'app-shared/types/ResourceAdm';
+
+/**
+ * Filter the list based on what is typed in the search box
+ */
+export const filterTableData = (searchValue: string, list: ResourceListItem[]): ResourceListItem[] => {
+  const searchValueLower = searchValue.toLocaleLowerCase();
+
+  return list.filter((resource: ResourceListItem) => {
+    const titles = Object.values(resource.title).map((title) => title.toLocaleLowerCase());
+    return titles.some((titleString) => titleString.includes(searchValueLower));
```

Comment:
Suggest to use the same naming on titles as the searchStringLower.
```suggestion
    return titles.some((titleLower) => titleLower.includes(searchStringLower));
```